src/Components/ToggleEdit.js

```diff
@@ -0,0 +1,23 @@
+import EditForm from "./Forms/EditForm";
+import Form from "./Forms/Form";
+const ToggleEdit = (props) => {
+  return (
+    <>
+      {props.edit ? (
+        <EditForm
+          handleEditFormSubmit={props.handleEditFormSubmit}
+          setEdit={props.setEdit}
+          currentTodo={props.currentTodo}
+          handleEditInput={props.handleEditInput}
+        />
+      ) : (
+        <Form
+          handleFormSubmit={props.handleFormSubmit}
+          todo={props.todo}
+          handleInputChange={props.handleInputChange}
+        />
```

Comment:
if it's edit then you should editForm otherwise you show Form ? isn't it going to be nicer if you use if and else instead of shorthand in the middle of JSX ?